server-spi-private/src/main/java/org/keycloak/authorization/policy/provider/PolicyProviderFactory.java

```diff
@@ -38,6 +38,10 @@ default boolean isInternal() {
         return false;
     }
 
+    default boolean isEnabled() {
```

Comment:
See above we already have support to enable/disable providers - see https://github.com/keycloak/keycloak/blob/master/services/src/main/java/org/keycloak/authentication/authenticators/browser/ScriptBasedAuthenticatorFactory.java#L155